File: aiJ-client/assets/Script/room/handler/HeroSceneResponseHandler.ts
import {AiJ} from "../../ws/AiJ";
import FireKit from "../../fire/FireKit";
import AppConfig from "../../AppConfig";

/**
 * 游戏场景
 */
export default class HeroSceneResponseHandler extends AiJ.ResponseHandler {

    constructor() {
        super();
    }

    handler(aiJWs: AiJ.AiJWs, response: AiJ.Response) {
        FireKit.use(AppConfig.GAME_FIRE).fire("hero_scene", response);
    }

}

File: aiJ-mahjong/src/main/java/com/xiyoufang/aij/mahjong/tester/HeroSceneResponseHandler.java
package com.xiyoufang.aij.mahjong.tester;

import com.neovisionaries.ws.client.WebSocket;
import com.xiyoufang.aij.room.response.HeroSceneResponse;
import com.xiyoufang.aij.room.tester.ResponseHandler;
import com.xiyoufang.aij.room.tester.TesterHero;

/**
 * Created by 席有芳 on 2018-12-25.
 *
 * @author 席有芳
 */
public class HeroSceneResponseHandler extends ResponseHandler<HeroSceneResponse> {

    public HeroSceneResponseHandler() {
        super(HeroSceneResponse.class);
    }

    /**
     * 转换后的对象Handler
     *
     * @param testerHero testerHero
     * @param response   response
     * @param webSocket  webSocket
     */
    @Override
    protected void doHandle(TesterHero testerHero, HeroSceneResponse response, WebSocket webSocket) {
        
    }
}
